src/helpers.tsx

```diff
@@ -1,29 +1,50 @@
 /* eslint-disable import/prefer-default-export */
+import { logger } from '@storybook/client-logger';
 import { WrapperSettings } from '@storybook/addons';
 import { DEFAULT_PADDING } from './constants';
 
-type Option = {
-  name: string;
-  value: string;
-  default?: boolean;
+type Value = { name: string; value: string };
+
+type Values = { [key: string]: Value };
```

Comment:
hey @rbardini, yeah when I was working over the actual implementation I noticed it too. The couple reasons I didn't went trough that implementation before:

1. I was still under the object sort of line of thought since previous discussions.
2. I liked a bit more the idea of having a defined key to use per entry as a filter value to set which is the default. Thinking maybe to avoid possible clashes or difficulties with verbose names. Still I don't know how often it may happen or if would actually be an use case. 
3. Keys used to override default are shorter for users. 

e.g: 
`{name: 'Extra Large Mobile', value: '42px'}`, then the consumer to override in other places will need the name. Which could be solve by:
1. Keeping the object notation
2. Switching to array and add a key to the entry `{key: 'xl-mobile', name: 'Extra Large Mobile', value: '42px'}`
3. Suggesting the consumer trough docs to use constants so they can reuse it everywhere else. 

Please, let me know what do you think? :)

Reply:
hey @rbardini, I decided to work on a proposal for you on the same line of thought you suggested, with a minor tweak as I mentioned in the option 2. Please let me know what do you think? :)